content/en/_index.html

```diff
@@ -102,83 +94,76 @@ <h5 class="card-title text-white section-head">Model serving</h5>
+          <h5 class="card-title text-white section-head">Model Serving</h5>
           <p class="card-text text-white">
-			<a target="_blank" rel="noopener" href="/docs/components/pipelines/introduction/" >Kubeflow Pipelines</a> is a comprehensive solution for deploying
-            and managing end-to-end ML workflows. Use Kubeflow Pipelines for rapid and reliable experimentation.
-            You can schedule and compare runs, and examine detailed reports on each run.
+            <a href="https://kserve.github.io/website/" target="_blank" rel="noopener" >KServe</a> (previously <a href="/docs/external-add-ons/kserve/kserve/" target="_blank" rel="noopener" >KFServing</a>) aims to solve production model serving on Kubernetes by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and ONNX.
           </p>
         </div>
       </div>
       <div class="card border-primary-dark">
         <img
-          src="/docs/images/logos/MultiPlatform.png"
-		  class="card-img-top"
-		  draggable="false"
+          src="/docs/images/logos/tensorflow-pytorch.png"
+          class="card-img-top"
+          draggable="false"
           style="padding: 2rem;"
-          alt="Multiple Framework Logos"
+          alt="TensorFlow + PyTorch Logo"
         />
         <div class="card-body bg-primary-dark">
-          <h5 class="card-title text-white section-head">Multi-framework</h5>
+          <h5 class="card-title text-white section-head">Model Training</h5>
           <p class="card-text text-white">
-            Our development plans extend beyond TensorFlow. We're working
-			hard to extend the support of 
-			<a target="_blank" rel="noopener" href="/docs/components/training/pytorch/">PyTorch</a>, 
-			<a target="_blank" rel="noopener" href="/docs/components/training/mxnet/"> Apache MXNet</a>, 
-		  	<a target="_blank" rel="noopener" href="/docs/components/training/mpi/"> MPI</a>,
-		  	<a target="_blank" rel="noopener" href="/docs/components/training/xgboost/"> XGBoost</a>,
-			<a target="_blank" rel="noopener" href="/docs/components/training/chainer/"> Chainer</a>, and more. We
-			also integrate with 
-			<a target="_blank" rel="noopener" href="https://istio.io/">Istio</a> and 
-			<a target="_blank" rel="noopener" href="https://www.getambassador.io/">Ambassador</a> for ingress, 
-			<a target="_blank" rel="noopener" href="https://nuclio.io/">Nuclio</a> as a
-            fast multi-purpose serverless framework, and <a target="_blank" rel="noopener" href="https://www.pachyderm.io/">Pachyderm</a> for managing
-            your data science pipelines.
+            <a href="/docs/components/training/" target="_blank" rel="noopener" >Kubeflow Training Operators</a>
+            provide a unified, Kubernetes interface for training machine learning models on multiple frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.
```

Comment:
```suggestion
            provide a unified, Kubernetes interface for cost efficient and large scale distributed training machine learning models on multiple frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.
```

cc @kubeflow/wg-training-leads @kuizhiqing @tenzen-y

Reply:
I would suggest to say also HPC. We can also run HPC tasks not ML Training on MPIJob.

Reply:
@tenzen-y This card focused on "Model Training" and is trying to present a story about how Kubeflow enables Model Training on Kubernetes.

As this is the front page, we really don't want to overload the reader.

Reply:
@andreyvelich How do you feel about this?

> Kubeflow Training Operator is a unified interface for model training on Kubernetes. It helps you run distributed training jobs for popular frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.

Reply:
> This card focused on "Model Training" and is trying to present a story about how Kubeflow enables Model Training on Kubernetes.
> 
> As this is the front page, we really don't want to overload the reader.

It makes sense. Avoiding to say many details would be better.

Reply:
"""
Kubeflow Training Operator is a unified interface for model training on Kubernetes. It helps you to run scalable distributed training jobs and do fine-tuning for popular frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.
"""
If we want to mention fine-tuning as well.

Reply:
It could do with a break in the middle, so I landed on:

```
Kubeflow Training Operator is a unified interface for model training on Kubernetes. 
It runs scalable and distributed training jobs for popular frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.
```

Reply:
@andreyvelich It could do with a break in the middle, so I landed on:

> Kubeflow Training Operator is a unified interface for model training on Kubernetes. It runs scalable and distributed training jobs for popular frameworks including PyTorch, TensorFlow, MPI, MXNet, PaddlePaddle, and XGBoost.

If you are happy, lets get this PR merged, and you can update it however you want longer term.